students/jisoo/westagram/users/views.py

```diff
@@ -1,10 +1,25 @@
+
 import json
 
 from django.http import JsonResponse
 from django.views import View
```

Comment:
```suggestion
from django.http  import JsonResponse
from django.views import View
```
import 기준으로 정렬해 주세요.